### ISSUE TITLE
Segfault running first example

### ISSUE THREAD

**Issue by dpsanders** [JuliaReach/RangeEnclosures.jl#148]
Trying to run the first example in the docs gives
```jl
julia> f(x) = -x^3/6 + 5x
f (generic function with 1 method)

julia> dom = 1 .. 4

[29689] signal (11.2): Segmentation fault: 11
in expression starting at none:0
mpfr_get_str at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/libmpfr.6.dylib (unknown line)
sprnt_fp at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/libmpfr.6.dylib (unknown line)
mpfr_vasnprintf_aux at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/libmpfr.6.dylib (unknown line)
mpfr_snprintf at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/libmpfr.6.dylib (unknown line)
round_string at /Users/dsanders/.julia/packages/IntervalArithmetic/dhnJB/src/display.jl:161
round_string at /Users/dsanders/.julia/packages/IntervalArithmetic/dhnJB/src/display.jl:172 [inlined]
basic_representation at /Users/dsanders/.julia/packages/IntervalArithmetic/dhnJB/src/display.jl:190
representation at /Users/dsanders/.julia/packages/IntervalArithmetic/dhnJB/src/display.jl:279 [inlined]
representation at /Users/dsanders/.julia/packages/IntervalArithmetic/dhnJB/src/display.jl:279 [inlined]
show at /Users/dsanders/.julia/packages/IntervalArithmetic/dhnJB/src/display.jl:339 [inlined]
show at ./multimedia.jl:47
unknown function (ip: 0x12b1dc0b3)
ijl_apply_generic at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
#15 at /Users/dsanders/.julia/packages/OhMyREPL/gIS2Z/src/output_prompt_overwrite.jl:23
unknown function (ip: 0x12b194593)
ijl_apply_generic at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
with_repl_linfo at /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-honeycrisp-R17H3W25T9.0/build/default-honeycrisp-R17H3W25T9-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:557
display at /Users/dsanders/.julia/packages/OhMyREPL/gIS2Z/src/output_prompt_overwrite.jl:6 [inlined]
display at /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-honeycrisp-R17H3W25T9.0/build/default-honeycrisp-R17H3W25T9-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:281 [inlined]
display at ./multimedia.jl:340
unknown function (ip: 0x12b18452b)
ijl_apply_generic at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
jl_f__call_latest at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
print_response at /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-honeycrisp-R17H3W25T9.0/build/default-honeycrisp-R17H3W25T9-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:0
#57 at /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-honeycrisp-R17H3W25T9.0/build/default-honeycrisp-R17H3W25T9-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:287
jfptr_YY.57_60621 at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/sys.dylib (unknown line)
ijl_apply_generic at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
with_repl_linfo at /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-honeycrisp-R17H3W25T9.0/build/default-honeycrisp-R17H3W25T9-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:557
jfptr_with_repl_linfo_60011 at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/sys.dylib (unknown line)
ijl_apply_generic at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
print_response at /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-honeycrisp-R17H3W25T9.0/build/default-honeycrisp-R17H3W25T9-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:285
do_respond at /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-honeycrisp-R17H3W25T9.0/build/default-honeycrisp-R17H3W25T9-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:899
jfptr_do_respond_60122 at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/sys.dylib (unknown line)
ijl_apply_generic at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
#24 at /Users/dsanders/.julia/packages/OhMyREPL/gIS2Z/src/repl.jl:252
ijl_apply_generic at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
jl_f__call_latest at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
#invokelatest#2 at ./essentials.jl:816 [inlined]
invokelatest at ./essentials.jl:813 [inlined]
#27 at /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-honeycrisp-R17H3W25T9.0/build/default-honeycrisp-R17H3W25T9-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/LineEdit.jl:1603
unknown function (ip: 0x12b10c08f)
ijl_apply_generic at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
prompt! at /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-honeycrisp-R17H3W25T9.0/build/default-honeycrisp-R17H3W25T9-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/LineEdit.jl:2740
run_interface at /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-honeycrisp-R17H3W25T9.0/build/default-honeycrisp-R17H3W25T9-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/LineEdit.jl:2642
jfptr_run_interface_60632 at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/sys.dylib (unknown line)
ijl_apply_generic at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
run_frontend at /Users/julia/.julia/scratchspaces/a66863c6-20e8-4ff4-8a62-49f30b1f605e/agent-cache/default-honeycrisp-R17H3W25T9.0/build/default-honeycrisp-R17H3W25T9-0/julialang/julia-release-1-dot-9/usr/share/julia/stdlib/v1.9/REPL/src/REPL.jl:1300
#62 at ./task.jl:514
jfptr_YY.62_60045 at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/sys.dylib (unknown line)
ijl_apply_generic at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
start_task at /Users/dsanders/.julia/juliaup/julia-1.9.2+0.aarch64.apple.darwin14/lib/julia/libjulia-internal.1.9.dylib (unknown line)
Allocations: 15129869 (Pool: 15112113; Big: 17756); GC: 22
Segmentation fault: 11
```

This is on an Apple M2.

```

(@v1.9) pkg> st -m
Status `~/.julia/environments/v1.9/Manifest.toml`
  [621f4979] AbstractFFTs v1.5.0
  [398f06c4] AbstractLattices v0.2.1
  [1520ce14] AbstractTrees v0.4.4
⌃ [79e6a3ab] Adapt v3.6.2
  [27a7e980] Animations v0.4.1
  [c7e460c6] ArgParse v1.1.4
⌃ [4fba245c] ArrayInterface v7.4.11
  [69666777] Arrow v2.6.2
  [31f734f8] ArrowTypes v2.2.2
⌅ [67c07d97] Automa v0.8.4
  [13072b0f] AxisAlgorithms v1.0.1
  [39de3d68] AxisArrays v0.4.7
  [6e4b80f9] BenchmarkTools v1.3.2
  [d1d4a3ce] BitFlags v0.1.7
  [c3b6d118] BitIntegers v0.3.1
  [e1450e63] BufferedStreams v1.2.1
⌅ [fa961155] CEnum v0.4.2
⌅ [96374032] CRlibm v0.8.0
⌃ [d360d2e6] ChainRulesCore v1.16.0
  [da1fd8a2] CodeTracking v1.3.5
⌃ [5ba52731] CodecLz4 v0.4.0
⌃ [944b1d66] CodecZlib v0.7.2
⌅ [6b39b394] CodecZstd v0.7.2
  [a2cac450] ColorBrewer v0.4.0
  [35d6a980] ColorSchemes v3.24.0
  [3da002f7] ColorTypes v0.11.4
⌅ [c3611d14] ColorVectorSpace v0.9.10
  [5ae59095] Colors v0.12.10
  [861a8166] Combinatorics v1.0.2
  [bbf7d656] CommonSubexpressions v0.3.0
  [34da2185] Compat v4.10.0
  [f0e56b4a] ConcurrentUtilities v2.2.1
  [88353bc9] ConfParser v0.1.2
  [5218b696] Configurations v0.17.6
  [187b0558] ConstructionBase v1.5.4
  [d38c429a] Contour v0.6.2
  [a8cc5b0e] Crayons v4.1.1
  [9a962f9c] DataAPI v1.15.0
  [864edb3b] DataStructures v0.18.15
  [e2d170a0] DataValueInterfaces v1.0.0
  [927a84f5] DelaunayTriangulation v0.8.8
  [8bb1440f] DelimitedFiles v1.9.1
  [163ba53b] DiffResults v1.1.0
  [b552c78f] DiffRules v1.15.1
⌃ [31c24e10] Distributions v0.24.18
  [ffbed154] DocStringExtensions v0.9.3
  [4e289a0a] EnumX v1.0.4
  [90fa49ef] ErrorfreeArithmetic v0.5.2
⌃ [429591f6] ExactPredicates v2.2.3
  [460bff9d] ExceptionUnwrapping v0.1.9
  [e2ba6199] ExprTools v0.1.10
  [55351af7] ExproniconLite v0.10.4
⌃ [411431e0] Extents v0.1.1
  [c87230d0] FFMPEG v0.4.1
  [7a1cc6ca] FFTW v1.7.1
⌅ [fa42c844] FastRounding v0.2.0
  [5789e2e9] FileIO v1.16.1
⌅ [1a297f60] FillArrays v0.11.9
  [6a86dc24] FiniteDiff v2.21.1
  [fb4d412d] FixedPointDecimals v0.4.3
  [53c48c17] FixedPointNumbers v0.8.4
  [59287772] Formatting v0.4.2
  [f6369f11] ForwardDiff v0.10.36
  [b38be410] FreeType v4.1.0
  [663a7486] FreeTypeAbstraction v0.10.0
⌃ [fb4132e2] FuzzyCompletions v0.5.2
  [f7f18e0c] GLFW v3.4.1
  [e9467ef8] GLMakie v0.8.11
  [46192b85] GPUArraysCore v0.1.5
  [28b8d3ca] GR v0.72.10
  [cf35fbd7] GeoInterface v1.3.2
  [5c1252a2] GeometryBasics v0.4.9
  [a2bd30eb] Graphics v1.1.2
  [3955a311] GridLayoutBase v0.9.2
  [42e2da0e] Grisu v1.0.2
  [cd3eb016] HTTP v1.10.0
  [2cbf6e9d] HardSphereDynamics v0.1.3 `~/Dropbox/packages/HardSphereDynamics`
  [ac1192a8] HypertextLiteral v0.9.4
  [2803e5a7] ImageAxes v0.6.11
⌃ [c817782e] ImageBase v0.1.5
⌅ [a09fc81d] ImageCore v0.9.4
  [82e4d734] ImageIO v0.6.7
  [bc367c6b] ImageMetadata v0.9.9
  [9b13fd28] IndirectArrays v1.0.0
  [d25df0c9] Inflate v0.1.4
  [842dd82b] InlineStrings v1.4.0
  [18e54dd8] IntegerMathUtils v0.1.2
  [a98d9a8b] Interpolations v0.14.7
⌅ [d1acc4aa] IntervalArithmetic v0.17.8
⌃ [138f1668] IntervalConstraintProgramming v0.9.1
  [15111844] IntervalContractors v0.4.7
  [d2bf35a9] IntervalRootFinding v0.5.11
⌃ [8197267c] IntervalSets v0.7.7
  [3587e190] InverseFunctions v0.1.12
⌅ [92d709cd] IrrationalConstants v0.1.1
  [f1662d9f] Isoband v0.1.1
  [c8e1da08] IterTools v1.8.0
  [82899510] IteratorInterfaceExtensions v1.0.0
⌃ [1019f520] JLFzf v0.1.5
  [692b3bcd] JLLWrappers v1.5.0
  [682c06a0] JSON v0.21.4
  [0f8b85d8] JSON3 v1.13.2
  [b835a17e] JpegTurbo v0.1.4
  [aa1ae85d] JuliaInterpreter v0.9.26
  [70703baa] JuliaSyntax v0.4.6
  [5ab0869b] KernelDensity v0.6.7
  [b964fa9f] LaTeXStrings v1.3.0
  [23fbe1c1] Latexify v0.16.1
  [0e77f7df] LazilyInitializedFields v1.2.1
  [8cdb02fc] LazyModules v0.3.1
  [9c8b4983] LightXML v0.9.0
  [d3d80556] LineSearches v7.2.0
  [9b3f67b0] LinearAlgebraX v0.1.12
  [2ab3a3ac] LogExpFunctions v0.3.26
  [e6f89c97] LoggingExtras v1.0.3
  [6f1432cf] LoweredCodeUtils v2.3.0
  [6c6e2e6c] MIMEs v0.1.4
  [1914dd2f] MacroTools v0.5.11
  [ee78f7c6] Makie v0.19.11
  [20f20a25] MakieCore v0.6.8
  [36869731] Malt v1.1.0
  [dbb5928d] MappedArrays v0.4.2
⌅ [7eb4fadd] Match v1.2.0
  [0a4f8689] MathTeXEngine v0.5.6
  [739be429] MbedTLS v1.1.7
  [442fdcdd] Measures v0.3.2
  [7269a6da] MeshIO v0.4.10
  [e1d29d7a] Missings v1.1.0
  [78c3b35d] Mocking v0.7.7
  [66fc600b] ModernGL v1.1.7
  [7475f97c] Mods v1.3.3
  [e94cdb99] MosaicViews v0.3.4
  [99f44e22] MsgPack v1.2.0
  [3b2b4ff1] Multisets v0.4.4
  [d41bc354] NLSolversBase v7.8.3
  [77ba4419] NaNMath v1.0.2
  [f09324ee] Netpbm v1.1.1
  [510215fc] Observables v0.5.4
  [6fe1bfb0] OffsetArrays v1.12.10
  [5fb14364] OhMyREPL v0.5.23
  [52e1d378] OpenEXR v0.3.2
  [4d8831e6] OpenSSL v1.4.1
  [429524aa] Optim v1.7.8
  [bac558e1] OrderedCollections v1.6.2
⌃ [90014a1f] PDMats v0.11.25
⌃ [f57f5aa1] PNGFiles v0.4.0
  [19eb6ba3] Packing v0.5.0
  [5432bcbf] PaddedViews v0.5.12
  [d96e819e] Parameters v0.12.3
  [69de0a69] Parsers v2.7.2
  [2ae35dd2] Permutations v0.4.17
  [b98c9c47] Pipe v1.3.0
  [eebad327] PkgVersion v0.3.3
  [ccf2f8ad] PlotThemes v3.1.0
  [995b91a9] PlotUtils v1.3.5
  [91a5bcdd] Plots v1.39.0
⌃ [c3e4b0f8] Pluto v0.19.29
  [647866c9] PolygonOps v0.1.2
⌅ [f27b6e38] Polynomials v3.2.13
  [2dfb63ee] PooledArrays v1.4.3
  [85a6dd25] PositiveFactorizations v0.2.4
  [91cefc8d] PrecompileSignatures v3.0.3
  [aea7be01] PrecompileTools v1.2.0
  [21216c6a] Preferences v1.4.1
  [27ebfcd6] Primes v0.5.4
  [92933f4c] ProgressMeter v1.9.0
  [3349acd9] ProtoBuf v1.0.14
  [4b34888f] QOI v1.0.0
  [1fd47b50] QuadGK v2.9.1
  [9c30249a] RAI v0.2.8
  [b3c3ace0] RangeArrays v0.3.2
  [1b4d18b6] RangeEnclosures v0.2.2
  [c84ed2f1] Ratios v0.4.5
  [379f33d0] ReachabilityBase v0.2.0
  [3cdcf5f2] RecipesBase v1.3.4
  [01d81517] RecipesPipeline v0.6.12
  [189a3867] Reexport v1.2.2
  [2792f1a3] RegistryInstances v0.1.0
  [05181044] RelocatableFolders v1.0.1
  [ae029012] Requires v1.3.0
⌃ [295af30f] Revise v3.5.6
  [286e9d63] RingLists v0.2.8
  [79098fc4] Rmath v0.7.1
  [5eaf0fd0] RoundingEmulator v0.2.1
  [6c6a2e73] Scratch v1.2.0
  [91c51154] SentinelArrays v1.4.0
  [3cc68bcd] SetRounding v0.2.1
  [efcf1570] Setfield v1.1.1
  [65257c39] ShaderAbstractions v0.4.0
  [992d4aef] Showoff v1.0.3
  [73760f76] SignedDistanceFields v0.4.0
  [777ac1f9] SimpleBufferStream v1.1.0
  [55797a34] SimpleGraphs v0.8.4
  [ec83eff0] SimplePartitions v0.3.0
  [cc47b68c] SimplePolynomials v0.2.14
  [a6525b86] SimpleRandom v0.3.1
  [699a6c99] SimpleTraits v0.9.4
  [45858cf5] Sixel v0.1.3
⌃ [a2af1166] SortingAlgorithms v1.1.1
⌅ [276daf66] SpecialFunctions v1.8.8
⌃ [c5dd0088] StableHashTraits v1.0.1
  [cae243ae] StackViews v0.1.1
  [90137ffa] StaticArrays v1.6.5
  [1e83bf80] StaticArraysCore v1.4.2
  [82ae8749] StatsAPI v1.7.0
⌅ [2913bbd2] StatsBase v0.33.21
⌅ [4c63d2b9] StatsFuns v0.9.18
  [09ab397b] StructArrays v0.6.16
  [856f2bd8] StructTypes v1.10.0
  [dc5dba14] TZJData v1.0.0+2023c
  [3783bdb8] TableTraits v1.0.1
⌃ [bd369af6] Tables v1.11.0
  [62fd8b95] TensorCore v0.1.1
  [b718987f] TextWrap v1.0.1
⌅ [731e570b] TiffImages v0.6.8
  [f269a46b] TimeZones v1.13.0
⌅ [3bb67fe8] TranscodingStreams v0.9.13
  [410a4b4d] Tricks v0.1.8
  [981d1d27] TriplotBase v0.1.0
  [9d95972d] TupleTools v1.4.3
⌃ [5c2747f8] URIs v1.5.0
  [3a884ed6] UnPack v1.0.2
  [1cfade01] UnicodeFun v0.4.1
  [1986cc42] Unitful v1.17.0
  [45397f5d] UnitfulLatexify v1.6.3
  [41fe7b60] Unzip v0.2.0
  [efce3f68] WoodburyMatrices v0.5.5
  [6e34b625] Bzip2_jll v1.0.8+0
  [83423d85] Cairo_jll v1.16.1+1
  [5ae413db] EarCut_jll v2.2.4+0
  [2702e6a9] EpollShim_jll v0.0.20230411+0
  [2e619515] Expat_jll v2.5.0+0
⌃ [b22a6f82] FFMPEG_jll v4.4.2+2
  [f5851436] FFTW_jll v3.3.10+0
  [a3f928ae] Fontconfig_jll v2.13.93+0
  [d7e528f0] FreeType2_jll v2.13.1+0
  [559328eb] FriBidi_jll v1.0.10+0
  [0656b61e] GLFW_jll v3.3.8+0
  [d2c73de3] GR_jll v0.72.10+0
  [78b55507] Gettext_jll v0.21.0+0
  [7746bdde] Glib_jll v2.76.5+0
  [3b182d85] Graphite2_jll v1.3.14+0
  [2e76f6c2] HarfBuzz_jll v2.8.1+1
  [905a6f67] Imath_jll v3.1.7+0
  [1d5cc7b8] IntelOpenMP_jll v2023.2.0+0
  [aacddb02] JpegTurbo_jll v2.1.91+0
  [c1c5ebd0] LAME_jll v3.100.1+0
  [88015f11] LERC_jll v3.0.0+1
  [1d63c593] LLVMOpenMP_jll v15.0.4+0
  [dd4b983a] LZO_jll v2.10.1+0
⌅ [e9f186c6] Libffi_jll v3.2.2+1
  [d4300ac3] Libgcrypt_jll v1.8.7+0
  [7e76a0d4] Libglvnd_jll v1.6.0+0
  [7add5ba3] Libgpg_error_jll v1.42.0+0
  [94ce4f54] Libiconv_jll v1.17.0+0
  [4b2f31a3] Libmount_jll v2.35.0+0
  [89763e89] Libtiff_jll v4.5.1+1
  [38a345b3] Libuuid_jll v2.36.0+0
  [5ced341a] Lz4_jll v1.9.4+0
  [856f044c] MKL_jll v2023.2.0+0
  [e7412a2a] Ogg_jll v1.3.5+1
  [18a262bb] OpenEXR_jll v3.1.4+0
⌅ [458c3c95] OpenSSL_jll v1.1.23+0
  [efe28fd5] OpenSpecFun_jll v0.5.5+0
  [91d4177d] Opus_jll v1.3.2+0
  [30392449] Pixman_jll v0.42.2+0
  [c0090381] Qt6Base_jll v6.5.2+2
  [f50d1b31] Rmath_jll v0.4.0+0
  [a44049a8] Vulkan_Loader_jll v1.3.243+0
  [a2964d1f] Wayland_jll v1.21.0+1
  [2381bf8a] Wayland_protocols_jll v1.25.0+0
  [02c8fc9c] XML2_jll v2.11.5+0
  [aed1982a] XSLT_jll v1.1.34+0
  [ffd25f8a] XZ_jll v5.4.4+0
  [f67eecfb] Xorg_libICE_jll v1.0.10+1
  [c834827a] Xorg_libSM_jll v1.2.3+0
  [4f6342f7] Xorg_libX11_jll v1.8.6+0
  [0c0b7dd1] Xorg_libXau_jll v1.0.11+0
  [935fb764] Xorg_libXcursor_jll v1.2.0+4
  [a3789734] Xorg_libXdmcp_jll v1.1.4+0
  [1082639a] Xorg_libXext_jll v1.3.4+4
  [d091e8ba] Xorg_libXfixes_jll v5.0.3+4
  [a51aa0fd] Xorg_libXi_jll v1.7.10+4
  [d1454406] Xorg_libXinerama_jll v1.1.4+4
  [ec84b674] Xorg_libXrandr_jll v1.5.2+4
  [ea2f1a96] Xorg_libXrender_jll v0.9.10+4
  [14d82f49] Xorg_libpthread_stubs_jll v0.1.1+0
  [c7cfdc94] Xorg_libxcb_jll v1.15.0+0
  [cc61e674] Xorg_libxkbfile_jll v1.1.2+0
  [e920d4aa] Xorg_xcb_util_cursor_jll v0.1.4+0
  [12413925] Xorg_xcb_util_image_jll v0.4.0+1
  [2def613f] Xorg_xcb_util_jll v0.4.0+1
  [975044d2] Xorg_xcb_util_keysyms_jll v0.4.0+1
  [0d47668e] Xorg_xcb_util_renderutil_jll v0.3.9+1
  [c22f9ab0] Xorg_xcb_util_wm_jll v0.4.1+1
  [35661453] Xorg_xkbcomp_jll v1.4.6+0
  [33bec58e] Xorg_xkeyboard_config_jll v2.39.0+0
  [c5fb5394] Xorg_xtrans_jll v1.5.0+0
  [3161d3a3] Zstd_jll v1.5.5+0
  [35ca27e7] eudev_jll v3.2.9+0
⌅ [214eeab7] fzf_jll v0.29.0+0
  [1a1c6b14] gperf_jll v3.1.1+0
  [9a68df92] isoband_jll v0.2.3+0
  [a4ae2306] libaom_jll v3.4.0+0
  [0ac62f75] libass_jll v0.15.1+0
  [2db6ffa8] libevdev_jll v1.11.0+0
  [f638f0a6] libfdk_aac_jll v2.0.2+0
  [36db933b] libinput_jll v1.18.0+0
  [b53b4c65] libpng_jll v1.6.38+0
  [075b6546] libsixel_jll v1.10.3+0
  [f27f6e37] libvorbis_jll v1.3.7+1
  [009596ad] mtdev_jll v1.1.6+0
  [1270edf5] x264_jll v2021.5.5+0
  [dfaa095f] x265_jll v3.5.0+0
  [d8fb68d0] xkbcommon_jll v1.4.1+1
  [0dad84c5] ArgTools v1.1.1
  [56f22d72] Artifacts
  [2a0f44e3] Base64
  [8bf52ea8] CRC32c
  [ade2ca70] Dates
  [8ba89e20] Distributed
  [f43a241f] Downloads v1.6.0
  [7b1f6079] FileWatching
  [9fa8497b] Future
  [b77e0a4c] InteractiveUtils
  [4af54fe1] LazyArtifacts
  [b27032c2] LibCURL v0.6.3
  [76f85450] LibGit2
  [8f399da3] Libdl
  [37e2e46d] LinearAlgebra
  [56ddb016] Logging
  [d6f4376e] Markdown
  [a63ad114] Mmap
  [ca575930] NetworkOptions v1.2.0
  [44cfe95a] Pkg v1.9.2
  [de0858da] Printf
  [9abbd945] Profile
  [3fa0cd96] REPL
  [9a3f8284] Random
  [ea8e919c] SHA v0.7.0
  [9e88b42a] Serialization
  [1a1011a3] SharedArrays
  [6462fe0b] Sockets
  [2f01184e] SparseArrays
  [10745b16] Statistics v1.9.0
  [4607b0f0] SuiteSparse
  [fa267f1f] TOML v1.0.3
  [a4e569a6] Tar v1.10.0
  [8dfed614] Test
  [cf7118a7] UUIDs
  [4ec0a83e] Unicode
  [e66e0078] CompilerSupportLibraries_jll v1.0.5+0
  [deac9b47] LibCURL_jll v7.84.0+0
  [29816b5a] LibSSH2_jll v1.10.2+0
  [c8ffd9c3] MbedTLS_jll v2.28.2+0
  [14a3606d] MozillaCACerts_jll v2022.10.11
  [4536629a] OpenBLAS_jll v0.3.21+4
  [05823500] OpenLibm_jll v0.8.1+0
  [efcefdf7] PCRE2_jll v10.42.0+0
  [bea87d4a] SuiteSparse_jll v5.10.1+6
  [83775a58] Zlib_jll v1.2.13+0
  [8e850b90] libblastrampoline_jll v5.8.0+0
  [8e850ede] nghttp2_jll v1.48.0+0
  [3f19e933] p7zip_jll v17.4.0+0
Info Packages marked with ⌃ and ⌅ have new versions available, but those with ⌅ are restricted by compatibility constraints from upgrading. To see why use `status --outdated -m`
```

```
julia> versioninfo()
Julia Version 1.9.2
Commit e4ee485e909 (2023-07-05 09:39 UTC)
Platform Info:
  OS: macOS (arm64-apple-darwin22.4.0)
  CPU: 12 × Apple M2 Max
  WORD_SIZE: 64
  LIBM: libopenlibm
  LLVM: libLLVM-14.0.6 (ORCJIT, apple-m1)
  Threads: 1 on 8 virtual cores
Environment:
  JULIA_EDITOR = code
  JULIA_PKG_DEVDIR = /Users/dsanders/Dropbox/packages/
```

**Comment by dpsanders** [comment]
Just doing 
```
julia> using IntervalArithmetic

julia> 3..4
```
gives the segfault.

**Comment by dpsanders** [comment]
Fixed by upgrading to version 0.21.2 of IntervalArithmetic.jl
